mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -99,7 +99,10 @@ immediately, with more details in dedicated headers below:
 an existing pre-`5.0.0` resource before an `apply`, the plan will show an
 update adding your current `labels` values to `terraform_labels`. This change
 may result in a no-op update call to the API depending on the resource
```

Comment:
How about this?

> This change will result in a no-op update on most resources, which can be safely applied. However, some resources are immutable. Before `5.X.0` this would fail. After `5.X.0`, a plan that includes recreating the resource will be created. This may be applied to move forward with the upgrade but **will** recreate the resource if you do so. We plan to make a change for these resources that will be recreated that will compute the `terraform_labels` value as part of the upgrade, and will come out for affected resources in the following releases after 5.X.0.